### ISSUE TITLE
Explorer implementation doesn't work

### ISSUE THREAD

**Issue by AlencarGabriel** [oleg-shilo/codemap.vscode#28]
Hi, I was testing the new configuration to change the location of the View CodeMap (#27), when I realized that it was not working.

![image](https://user-images.githubusercontent.com/10109480/73738323-0d40cf00-4723-11ea-9bae-3e288a5eecf4.png)

![image](https://user-images.githubusercontent.com/10109480/73738356-1b8eeb00-4723-11ea-9c69-0e43aa2b5734.png)

<details><summary>Configuration</summary>

``` json
"codemap.prw": "C:\\Users\\g_santana\\AppData\\Roaming\\Code\\User\\mapper_prw.js",
"codemap.prx": "C:\\Users\\g_santana\\AppData\\Roaming\\Code\\User\\mapper_prw.js",
"codemap.textModeExpanded": false,
"codemap.textMode": false,
"codemap.location": "codemap",
```
</details>

**Comment by oleg-shilo** [comment]
Don't forget to reload VSCode.
I just have tested it:
![image](https://user-images.githubusercontent.com/16729806/73739871-2dac7000-479b-11ea-98ca-39e4ed370d23.png)
![image](https://user-images.githubusercontent.com/16729806/73739882-32712400-479b-11ea-88de-75b76f7be8b5.png)

Seems to work...

**Comment by AlencarGabriel** [comment]
Yes, I recharged VsCode as a guarantee.

As you can see in the prints, the view appears, but does not show the code structure.

**Comment by oleg-shilo** [comment]
You are right. Sorry I missed this point. 
The code change was contributed by @greatpate I will ask him to have a look. Hopefully be it's just a tiny configuration issue.

**Comment by AlencarGabriel** [comment]
Thankful! 

This option is really useful, when I saw the release notes I already wanted to configure.

Waiting for correction.

**Comment by greatpate** [comment]
Hi Oleg,
Sorry I haven't been with my computer for a few days. I think the issue is in the package.json where we have the commands
![image](https://user-images.githubusercontent.com/27994049/74297218-466be580-4d9a-11ea-9746-b57d5a4b24cb.png)
During my testing of the feature I must have had the explorer id set to codemap at one point and it not need to refresh.
I still don't have access to my computer unfortunately but hopefully my suggestion helps

**Comment by oleg-shilo** [comment]
Hey Daniel, thank you for stepping in. Ironically my bandwidth is not so great either. Just out of the hospital.
Though I managed to do a quick test.

The changes you suggested are logical but it looks like it's not enough. Even the code below does not help:
```

            "view/title": [
                {
                    "command": "codemap.refresh",
                    "when": "view == explorer",
                    "group": "navigation@2"
                }
            ],
            "view/item/context": [
                {
                    "command": "codemap.navigate_to_selected",
                    "when": "view == explorer && viewItem == file"
```

**Comment by oleg-shilo** [comment]
Done. Fixed in Release v1.10.4

**Comment by AlencarGabriel** [comment]
Very good! Now in Explorer view everything is working correctly.

Thanks for listening!
